consensus/ibft/signer/extra.go

```diff
@@ -20,6 +22,8 @@ var (
 	IstanbulExtraSeal = 65
 
 	zeroBytes = make([]byte, 32)
+
+	ErrRoundNumberOverflow = errors.New("round number is out of range for 64bit")
```

Comment:
Should we make this local (unexport it)?

Reply:
Fixed c2fee48aa934d3210d2de82b9de7d339769ea35e